static/js/components/search/FilterVisibilityToggle.js

```diff
@@ -64,15 +64,19 @@ export default class FilterVisibilityToggle extends SearchkitComponent {
     return false;
   };
 
+  isFilterSelected = (id: string): boolean => {
```

Comment:
Can you change `stayVisibleIfEmpty` to make use of this function since it looks like they overlap?